### ISSUE TITLE
Trident Hardware Issue - Test A Fail Error

### ISSUE THREAD

**Issue by daniellembecker** [Putnam-Lab/Lab_Management#39]
The Trident started to display a Test A Fail Error within the past few months. All troubleshooting options have been exhausted by @laurenzane @daniellembecker and Pierrick to troubleshoot and fix the issue on our own. We referenced the comprehensive [Neptune Trident Troubleshooting Guide](https://www.neptunesystems.com/getstarted/trident/troubleshooting/) to assist us with every possible option to fix this issue on our own. 

General notes on the issue:

The Trident is giving us an error of test A fail because when an alkalinity test is run, reagent A is only filling up just above the stir bar in the cuvette or less before being released. The reason we need to reach out to the company is that when I do the priming for just the sample and reagent A to see if there is any buildup or issues with the line, they both successfully fill up to the top of the cuvette. This indicates that there are no issues with the lines but it seems like the timing is off or something for the alkalinity test is off that is having it dispense the reagent A way too quickly during the process.

@daniellembecker scheduled a troubleshooting call with Neptune Systems on July 14th and from the conversation with all descriptive information provided, the resolution was that this is a hardware issue and the Trident will need to be sent in for repair. The Neptune representative (Morgan) asked @daniellembecker to check the buttons located on the white solenoids just above the T shape that is raised (see photo) in the back of the Trident to see if any of the buttons were stuck but they were not. Morgan then recommended one of three options:

We can have you send the Trident in and service it, which can be a 2-3 week turnaround time, or we have a cross ship option where we have you put down a deposit that we hold.  
We then send out a replacement Trident, and you send in the old one to us.  
We refund the deposit minus any out of warranty repair costs after we receive the old Trident back and check it in.

The next steps are to send the Trident out for repair with one of the three options above with @hputnam approval.

Neptune Systems Contact:

Morgan Wood
Agent, Technical Support
T: 408-275-2205

When you call, make sure to press 1 for technical support and 2 again for an already started issue, our ticket number is #234974

**Comment by daniellembecker** [comment]
Update: @daniellembecker sent an email to Morgan/Neptune Systems with the required information to begin a return merchandise authorization (RMA) for whichever option we choose to move forward with.

**Comment by daniellembecker** [comment]
Update: Neptune customer support has suggested we need to service and repair our Trident. There are multiple options with various costs on their [Trident Service Program](https://www.neptunesystems.com/tridentservice/) page. @hputnam if you don't mind looking over which option you prefer then a service form can be filled out

**Comment by hputnam** [comment]
Please do the Send and Trident Service. Thank you!

**Comment by hputnam** [comment]
[20220718_neptune_systems.pdf](https://github.com/Putnam-Lab/Lab_Management/files/9133792/20220718_neptune_systems.pdf)

**Comment by daniellembecker** [comment]
I filled out the RMA form [RMA_Trident.pdf](https://github.com/Putnam-Lab/Lab_Management/files/9144095/RMA_Trident.pdf)from Neptune and created a FedEx shipping label. Sent out today for two-day shipping, arriving at Neptune on July 21st.

Tracking number
777428533054

Pickup confirmation number
PVDA-3332

**Comment by hputnam** [comment]
Repaired Trident arrived URI 8/4/22. @daniellembecker @laurenzane can you please set this up next to the other one and test it? Then we will move it to the quarantine tank once we have cross compared it with the other Trident.

**Comment by daniellembecker** [comment]
@hputnam I can do this Monday when I go in

**Comment by laurenzane** [comment]
I opened the old Trident on Tuesday, August 25th and reagent B was missing the uptake tube. I called Neptune Systems on 8/25 and they agreed to send us a new one for free. A correction to a comment above, the help line for an already started issue is first 1 for technical support and **2** if you already have a ticket. I will update this again when the tube comes in and I can set up the Trident in the QT.
